packages/components/src/Progress.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+
+import { ThemeUICSSObject } from '@theme-ui/css'
+
+import { Box, BoxOwnProps } from './Box'
+import type { Assign, ForwardRef } from './types'
+
+export interface ProgressProps
+  extends Assign<React.ComponentPropsWithRef<'progress'>, BoxOwnProps> {}
+
+/**
+ * @see https://theme-ui.com/components/progress/
```

Comment:
Ideally would add the text description here?

Reply:
Dude, I can't believe you reviewed this the same day as I pushed :D

Reply:
How about this:

```suggestion
 * Horizontal `<progress />` bar
 * @see https://theme-ui.com/components/progress/
```

BTW, I noticed the docs are a bit silly here :D

![image](https://user-images.githubusercontent.com/15332326/146807021-84fd0b07-8a41-49e8-8c51-915d09cc4e9e.png)

Reply:
Lol yeah, need to take a pass through those. Noted!

Reply:
Fixed that separately in #2069.